src/test/java/org/gridsuite/sensitivityanalysis/server/SensitivityAnalysisControllerTest.java

```diff
@@ -748,10 +748,8 @@ public void stopTest() throws Exception {
             .andExpect(status().isOk());
 
         // stop sensitivity analysis
-        assertNotNull(output.receive(TIMEOUT, "sensitivityanalysis.run"));
```

Comment:
can you explain about this ?

Reply:
The consumed messages no longer appear on the output so we can't double consume them anymore (one by the server and one in the test). I checked and the workflow is exactly the same as before the springboot 3 migration only the message isn't there.